### ISSUE TITLE
Some kind of game.

### ISSUE THREAD

**Issue by aidanbdh** [aidanbdh/Group-Project-Ideas#2]
Make with unity

**Comment by ansonhwo** [comment]
If we wanted to do something related to websites we could recreate popular board games like Settlers of Catan/Ticket to Ride

**Comment by aidanbdh** [comment]
That would be fun :) What do you guys think @josephgutierrez  @thienmdoan ?

**Comment by josephgutierrez** [comment]
I need to check those games out, I have never heard of them before :)

**Comment by aidanbdh** [comment]
@ansonhwo  do you have those games? I have a bunch of random board games but not those ones.

**Comment by ansonhwo** [comment]
Yeah I have Settlers of Catan, it's a fairly competitive resource management game. It can get heated, haha. I've played Ticket to Ride, but I don't own it.

There are all sorts of board games out there, ranging from friendly ones like Sushi Go to really advanced ones like Food Chain Magnate. I would imagine that this kind of project would need to utilize SocketIO on the backend for real-time chat and turn-by-turn play.

**Comment by aidanbdh** [comment]
We should get together sometime to play some! We should also do some coop games as well. I have some and they're really fun, good for a break from competitiveness
